NuGet/Specs/Tealium.Maui.nuspec

```diff
@@ -16,33 +16,33 @@
     <tags>Tealium iOS Android</tags>
     <!-- Insert within the <metadata> element -->
     <dependencies>
-        <group targetFramework="net8.0-android33.0">
-            <dependency id="Xamarin.Kotlin.StdLib" version="[1.5,2.0)" />
-            <dependency id="Xamarin.KotlinX.Coroutines.Core" version="[1.5,2.0)" />
-            <dependency id="Xamarin.KotlinX.Coroutines.Android" version="[1.5,2.0)" />
+        <group targetFramework="net9.0-android34.0">
+            <dependency id="Xamarin.Kotlin.StdLib" version="[2.0,3.0)" />
+            <dependency id="Xamarin.KotlinX.Coroutines.Core" version="[2.0,3.0)" />
+            <dependency id="Xamarin.KotlinX.Coroutines.Android" version="[2.0,3.0)" />
```

Comment:
Are these coroutines versions correct? - seems that the latest is [1.9.0.2](https://www.nuget.org/packages/Xamarin.KotlinX.Coroutines.Android)